examples/nextjs-spa/.env

```
@@ -0,0 +1 @@
+ORY_SDK_URL = "https://ecstatic-cori-adbxpd2qln.projects.oryapis.com"
```

Point this to tunnel instead of the project directly.
```suggestion
ORY_SDK_URL = "http://localhost:4000"
```